### DISCUSSION TITLE
How can I changed the exchange-current density

### DISCUSSION THREAD

**Discussion by AmadeusTina** [pybamm-team/PyBaMM#1733] [Q&A] [answered]
Hi, everyone.
I want to know how to change exchange-current density in parameterset.
They are defined function and I don't know how to replace it with myself.

**Comment by brosaplanella** [comment, marked as answer]
Hi! You need to define it as a Python function of this form (with this particular inputs)
```python3
def your_electrolyte_exchange_current_density(c_e, c_s_surf, T):
```
and then you can pass your function to the parameters dictionary doing
```python3
param["Positive electrode exchange-current density [A.m-2]"] = exchange_current_density
```
(or with the negative one if that's the one you want to change.

**Reply by AmadeusTina** [reply]
Thank you!!!😊
